test/jdk/javax/swing/ButtonGroup/TestButtonGroupFocusTraversal.java

```diff
@@ -30,6 +30,7 @@
  * @run main TestButtonGroupFocusTraversal
  */
 
```

Comment:
We need to append this bugid as product code is changed for which this testcase is modified

Reply:
Done, please have a look.